Cargo.toml

```diff
@@ -22,22 +22,22 @@ encoding = ["encoding_rs"]
 
 [dependencies]
 futures = { version = "0.3.1", features = ["compat", "io-compat"] }
-http = "0.1.17"
+http = "0.2.1"
 log = { version = "0.4.7", features = ["kv_unstable"] }
 mime = "0.3.13"
 mime_guess = "2.0.3"
 serde = "1.0.97"
 serde_json = "1.0.40"
 serde_urlencoded = "0.6.1"
 url = "2.0.0"
-http-client = { version = "1.1.0", features = ["native_client"] }
+http-client = { git = "https://github.com/mythmon/http-client", branch = "isahc-0.9", features = ["native_client"] }
```

Comment:
Notably, this line will need to be updated to something like this, once my PR to http-client merges and is released.

```toml
http-client = { version = "2.0.0", features = ["native_client"] }
```